ansible_collections/arista/avd/molecule/eos_cli_config_gen_v4.0/inventory/host_vars/ip-extended-community-lists.yml

```diff
@@ -1,11 +1,13 @@
 ### IP extended community lists ###
 
 ip_extcommunity_lists:
-  TEST1:
-    - type: permit
-      extcommunities: "65000:65000"
-    - type: deny
-      extcommunities: "65002:65002"
-  TEST2:
-    - type: deny
-      extcommunities: "65001:65001"
+  - name: TEST1
+    types:
```

Comment:
For the example, it is great to see the updated filter in function here. But please remove everything else than the filter from this PR.